lib/tether/tether.go

```diff
@@ -133,6 +136,15 @@ func (t *tether) setup() error {
 		}
 	}
 
+	// Create PID file for tether
+	paths := strings.Split(os.Args[0], "/")
+	err = ioutil.WriteFile(fmt.Sprintf("%s/%s.pid", PIDFileDir, paths[len(paths)-1]),
```

Comment:
@hmahmood Per our discussion, we may be moving vicadmin to be its own service not spawned by the tether.  In that event, it does make sense to create a PID file for the tether.